engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -2163,27 +2262,46 @@ void priest_t::generate_apl_shadow()
+                    "Only use Void Bolt if Devouring Plague doesn't need refreshed." );
   main->add_call_action_list( cds );
-  main->add_talent( this, "Damnation", "target_if=!variable.all_dots_up", "Prefer to use Damnation ASAP if any DoT is not up" );
+  main->add_talent( this, "Damnation", "target_if=!variable.all_dots_up",
+                    "Prefer to use Damnation ASAP if any DoT is not up" );
   main->add_action( this, "Devouring Plague", "target_if=(refreshable|insanity>75)&!cooldown.power_infusion.up",
-                      "Make sure you don't use Devouring Plague if you are trying to build into Voidform." );
-  main->add_action( this, "Shadow Word: Death", "target_if=target.health.pct<20", "Use Shadow Word: Death if the target is about to die." );
-  main->add_talent( this, "Surrender to Madness", "target_if=target.time_to_die<25&buff.voidform.down", "Use Surrender to Madness on a target that is going to die at the right time." );
+                    "Make sure you don't use Devouring Plague if you are trying to build into Voidform." );
+  main->add_action( this, "Shadow Word: Death", "target_if=target.health.pct<20",
+                    "Use Shadow Word: Death if the target is about to die." );
+  main->add_talent( this, "Surrender to Madness", "target_if=target.time_to_die<25&buff.voidform.down",
+                    "Use Surrender to Madness on a target that is going to die at the right time." );
   main->add_talent( this, "Mindbender" );
-  main->add_talent( this, "Void Torrent", "target_if=variable.all_dots_up&!cooldown.void_eruption.up&target.time_to_die>4", "Use Void Torrent only if all DoTs are active and the target won't die during the channel." );
+  main->add_talent( this, "Void Torrent",
+                    "target_if=variable.all_dots_up&!cooldown.void_eruption.up&target.time_to_die>4",
+                    "Use Void Torrent only if all DoTs are active and the target won't die during the channel." );
   main->add_action( this, "Shadow Word: Death",
-                      "if=(!runeforge.painbreaker_psalm.equipped|(runeforge.painbreaker_psalm.equipped&variable.dots_up))&target.health.pct>30",
-                      "TODO see if this is worth even without Painbreaker Psalm" );
+                    "if=(!runeforge.painbreaker_psalm.equipped|(runeforge.painbreaker_psalm.equipped&variable.dots_up))"
+                    "&target.health.pct>30",
+                    "TODO see if this is worth even without Painbreaker Psalm" );
   main->add_talent( this, "Shadow Crash", "if=raid_event.adds.in>5&raid_event.adds.duration<20",
-                      "Use Shadow Crash on CD unless there are adds incoming." );
+                    "Use Shadow Crash on CD unless there are adds incoming." );
+  main->add_action(
+      this, "Mind Sear",
+      "target_if=spell_targets.mind_sear>1&buff.dark_thoughts.up,chain=1,interrupt_immediate=1,interrupt_if=ticks>=2" );
```

Comment:
Can we add comments to these more complex lines to the APL?

```suggestion
      "target_if=spell_targets.mind_sear>1&buff.dark_thoughts.up,chain=1,interrupt_immediate=1,interrupt_if=ticks>=2", "Use Mind Sear to consume any Dark Thought procs on AoE" );
```

Reply:
Added alongside a TODO statement to recent commit